storage/onpremise/mongodb/locals.tf

```diff
@@ -1,6 +1,6 @@
 locals {
-  node_selector_keys   = keys(var.mongodb.node_selector)
-  node_selector_values = values(var.mongodb.node_selector)
+  node_selector_keys   = length(keys(var.mongodb.node_selector)) == 0 ? keys(var.mongodb.node_selector) : []
+  node_selector_values = length(values(var.mongodb.node_selector)) == 0 ? values(var.mongodb.node_selector) : []
```

Comment:
Is this really useful?

Reply:
I got errors when deploying if node_selector was null or '{}' but it might have been due to bad default value definition